build/packages.targets

```diff
@@ -70,15 +70,17 @@
           However, our MSBuild integration tests use Microsoft.Build 16.8.0, which requires System.Threading.Tasks.Dataflow 4.9.0 (assembly version 4.9.3.0).
           To resolve runtime assembly binding failures, we'll downgrade the package from 4.11.1 to 4.9.0.
         -->
+        <PackageReference Update="Microsoft.VisualStudio.VCProjectEngine" Version="16.9.31025.104" />
```

Comment:
By removing condition for VS15, is build script stop supporting VS15?

Reply:
In theory, yes. I tested building for VS15 with this change, and, it didn't work. Failures indicate a .NET Core SDK issue

Reply:
If I leave the VS15 condition, it could work, but, we still need to address the .NET Core SDK Issue

Reply:
We depend on changes to CPS & the .NET project system that are available only in d16. I think we're a long time past "not supporting" d15 from our dev branch.  Our build scripts just haven't kept up.

Reply:
It's potentially a chain with some tail, but we should change our output location to `VS` only or something else. 
`VS15` referred to dev15. When we moved to dev16, we didn't change it because we didn't want to chase the tail. 
I think we have an opportunity to change it now.

Reply:
So then do we need `15` in `[ValidateSet(15, 16, 17)]` now?

Reply:
Since the script does not build for dev15, I will remove `15` in the set.